docs/code_ref/data.rst

```diff
@@ -1,8 +1,8 @@
+*******************
 Data (`sunpy.data`)
 *******************
 
-``sunpy.data`` contains ways to access sample data and small test files for
-running the sunpy test suite.
+`sunpy.data` contains ways to access sample data and small test files for running the **sunpy** test suite.
```

Comment:
I wonder if all of this should just be at the top level of the python file for every package instead of here.